test/html/multiple-imports.html

```diff
@@ -7,5 +7,6 @@
     Code distributed by Google as part of the polymer project is also
     subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
+<link rel="import" href="multiple-imports.html">
```

Comment:
This is tricky to represent in a test, but the symptom of the arbitrary injection order was easy enough to replicate by adding a self-reference in the file here.  I'll see if I can clear-up the issue in tests now.